File: app/src/main/java/com/viafourasample/src/utils/AndroidUtils.java
package com.viafourasample.src.utils;

import android.content.Context;
import android.util.DisplayMetrics;

public class AndroidUtils {
    public static float convertDpToPixel(float dp, Context context){
        return dp * ((float) context.getResources().getDisplayMetrics().densityDpi / DisplayMetrics.DENSITY_DEFAULT);
    }
}

File: app/src/main/java/com/viafourasample/src/utils/AndroidUtils.kt
package com.viafourasample.src.utils

import android.content.Context
import android.util.DisplayMetrics

object AndroidUtils {
    fun convertDpToPixel(dp: Float, context: Context): Float =
        dp * (context.resources.displayMetrics.densityDpi.toFloat() / DisplayMetrics.DENSITY_DEFAULT)
}
